idea/indexer/src/services/message.service.ts

```diff
@@ -71,7 +70,7 @@ export class MessageService {
     };
   }
 
-  public async create(messages: Message[]): Promise<IMessage[]> {
+  public async create(messages: Message[]): Promise<Message[]> {
     for (const m of messages) {
```

Comment:
What is the advantage use a single letter? It makes code more readable and what kind of quality improves if using a single letter? In my opinion, it makes more decreases readable quality for other people who will look at this code.

Reply:
It's good when code is readable and short as much as it possible. In this case it's absolutely clear what `m` means. I don't see any problem